packages/mrm-core/src/npm.js

```diff
@@ -117,7 +117,9 @@ function runNpm(deps, options = {}, exec) {
  * @param {Function} [exec]
  */
 function runYarn(deps, options = {}, exec) {
-	const add = options.dev ? ['add', '--dev'] : ['add'];
+	const add = options.dev
+		? ['add', '--dev', '--ignore-workspace-root-check']
```

Comment:
Could you add a comment with the explanation of the flag here too?

Reply:
Added comments in the doc block.